manifests/kustomize/hack/presubmit.sh

```diff
@@ -22,22 +22,33 @@ DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null && pwd)"
 TMP="$(mktemp -d)"
 
 pushd "${TMP}"
+
 # Install Kustomize
 KUSTOMIZE_VERSION=5.2.1
+# Remove existing kustomize if it exists
+if [ -f "/usr/local/bin/kustomize" ]; then
+    echo "Removing existing kustomize from /usr/local/bin"
+    sudo rm /usr/local/bin/kustomize
```

Comment:
This is danger

Reply:
Example:

Download everything into the created temp folder https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/hack/presubmit.sh#L22
then,
```
PATH="$TMP:$PATH"
```
In this case, your downloaded binaries will be available for use without breaking your local environment.